### DISCUSSION TITLE
support setting `config-path` in vscode settings

### DISCUSSION THREAD

**Discussion by universalmind303** [biomejs/biome-vscode#244] [Ideas]
i'd like to configure the command line `--config-path` arg via my settings json as I commonly work in a monorepo that has a single node package within a larger project `<root>/js` (or similar). So i'd like to configure my `<root>/.vscode/settings.json` to always point to `<root>/js/biome.json` instead of having to open vscode with `js` as the root.

**Comment by ematipico** [comment]
I don't think it's easily possible at the moment, especially for VSCode, but I know for a fact that we need this configuration.

**Comment by hrasekj** [comment]
I agree that this feature would be great addition. 

Also for me, i like to use biome with my custom config, but without need to create biome.json in repository i'm working on.

**Comment by matej-kaska** [comment]
I have the same problem. My projects have two folders in the root `frontend` and `backend` and my `biome.json` config is not being loaded. So I cannot really use the extension.
